modules/n1ql_fts/pages/index.adoc

```diff
@@ -0,0 +1,589 @@
+----
+
+Finally, go to Couchbase's Web Console and click on the *Search* menu. If everything is right you should see something similar to the following:
+
+image:001-index-created.png[Index Created Successfully]
+
+
+== Searching for Movies
+
+=== Building a basic search
+
+Let's start by creating a query which searches for *Star War*. Note that we are searching for *Star War* instead of *Star WarS*. The goal is to build a simple search that already has some tolerance for typos.
+
+Run the following query via web console:
+
+[source, SQL]
+----
+
+select m.original_title, SEARCH_SCORE() AS score  from movies m where search(m, 
+    {
+       "should": { 
+                   "disjuncts": [
+                        {"field":"original_title", "match": "Star War"}, 
+                        {"field":"original_title", "match": "Star War", "fuzziness":1}
+                    ]
+             
+       }
+    }
+) 
+order by score desc
+limit 20
+
+----
+
+It will probably take a long time to run. You can see why in the query plan:
+
+image:002-slow-query-plan.png[Slow Query Plan]
+
+If you pay attention to the plan above, you will notice that no index has been used even though we have an index already in place. 
+
+Open the index we created via *Search -> movies_shingle*, you should see something similar to the following:
+
+image:003-index.png[movies index]
+
+Note that this definition only index documents with the type *_class* equals to *com.cb.fts.sample.entities.Movie*, and that is why our index wasn't used by our first query. If we rerun the same query filtering the result by the document type, it will much faster than before:
+[source, SQL,]
+----
+
+select m.original_title, SEARCH_SCORE() AS score  from movies m where search(m, 
+    {
+       "should": { 
+                   "disjuncts": [
+                        {"field":"original_title", "match": "Star War"}, 
+                        {"field":"original_title", "match": "Star War", "fuzziness":1}
+                    ]
+             
+       }
+    }
+) 
+and m._class = "com.cb.fts.sample.entities.Movie" 
+order by score desc
+limit 20
+
+----
+
+The query plan now shows a step called *IndexFtsSearch* which means, as you might guess, that our index is now being used.
+
+image:004-index-being-used.png[Index used]
+
+Now, let's try to understand what is going on with our query:
+
+[source, JavaScript, linenums]
+----
+"disjuncts": [
+    {"field":"original_title", "match": "Star War"}, 
+    {"field":"original_title", "match": "Star War", "fuzziness":1}
+]
+
+----
+
+We start by creating a disjunction of two matches of *Star War* in a field called *original_title*. The first one is an exact match, and the second has a fuzziness of 1. A *disjunction* is similar to an *OR* clause in SQL.
+
+TIP: If you are not familiar with link:https://blog.couchbase.com/fuzzy-matching/[fuziness or Levenshtein Distance, check out this blog post.]
```

Comment:
Please change `fuziness` to `fuzziness`

(Also, this is just a stylistic thing, please put the full stop outside the link, like so: `].`)